### ISSUE TITLE
yume - Inappropriate ioctl for device error

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#263]
**Reported by bli on 20 Sep 2006 00:07 UTC**
```
[~](root@o0001)# rpm -q yume
yume-2.3-1
```

```
[~](root@o0001)# yume
You need to give some command

usage: yum [< update | install | info | remove | list |
    clean | provides | search | check-update | groupinstall |
    groupupdate | grouplist | groupinfo | groupremove |
    makecache | localinstall | erase | upgrade | whatprovides |
    localupdate | resolvedep | shell | deplist >

options:
  -h, --help            show this help message and exit
  -t, --tolerant        be tolerant of errors
  -C                    run entirely from cache, don't update cache
  -c [config file](options])      config file location
  -R [         maximum command wait time
  -d [debug level](minutes])      debugging output level
  -e [level](error)      error output level
  -y                    answer yes for all questions
  --version             show Yum version and exit
  --installroot=[ set install root
  --enablerepo=[repo](path])   enable one or more repositories (wildcards allowed)
  --disablerepo=[ disable one or more repositories (wildcards allowed)
  --exclude=[package](repo])   exclude package(s) by name or glob
  --obsoletes           enable obsoletes processing during updates
  --noplugins           disable Yum plugins
ERROR while executing "yum -c /tmp/fileqr56ho " : Inappropriate ioctl for device
```

**Comment by dikim33** [comment]
**Comment by bli on 20 Sep 2006 23:57 UTC**
I also get the following if I run `yume install some_package` and then hit ctrl-c:

```
Exiting on user cancel
ERROR while executing "yum -c /tmp/fileMsQvSX install mpiJava-modulefile.{x86_64,noarch}" : Inappropriate ioctl for device
```

**Comment by dikim33** [comment]
**Comment by efocht on 26 Sep 2006 21:02 UTC**
Reducing priority.
Problem is most probably a yum problem, not related to yume, at least I can't see where yume does ioctls.

**Comment by dikim33** [comment]
**Modified by jparpail on 11 Apr 2007 09:34 UTC**

**Comment by dikim33** [comment]
**Comment by jparpail on 20 Jul 2007 14:54 UTC**
Is it reproductible ?

**Comment by dikim33** [comment]
**Comment by efocht on 13 Sep 2007 22:53 UTC**
(In [6166]) Removed nasty "inappropriate ioctl" messages at yum exectution.
Fixed repoquery for yum version > 3.
Updating version to 2.5-1
closes #263
closes #394
